.github/workflows/ci.yml

```diff
@@ -15,10 +15,10 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        php-version: ['7.2', '7.4', '8.0', '8.1']
+        php-version: ['8.0', '8.1']
```

Comment:
Why is PHP MIN now 8? Where did 7.2/7.3/7.4 go? At least the latter would be useful to keep IMO
Either way, this needs to be synced with composer.json min as well then.

Reply:
This is explained in the attached issue.